### PR TITLE
Implements autoscroll within log view

### PR DESCRIPTION
**Pull request by onionhammer**
Addresses #992

image

### CODE REVIEW

**Comment by jkotalik** [comment]
ooooo let me try this out!

File: src/Microsoft.Tye.Hosting/Dashboard/Pages/Logs.razor [hunk 1]
```diff
@@ -51,13 +71,30 @@ else
         base.OnInitialized();
     }
 
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        if (_autoScroll)
+            await ScrollToBottom();
```
Comment (jkotalik) [inline src/Microsoft.Tye.Hosting/Dashboard/Pages/Logs.razor:77]:
```suggestion
        {
            await ScrollToBottom();
        }
```

File: src/Microsoft.Tye.Hosting/wwwroot/js/logview.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/** Scroll to the bottom of the input element */
+function logviewScrollToBottom(elementId) {
```
Comment (jkotalik) [inline src/Microsoft.Tye.Hosting/wwwroot/js/logview.js:2]: It looks like my ad-blocker and or browser is blocking this script by default.

Comment (jkotalik) [reply src/Microsoft.Tye.Hosting/wwwroot/js/logview.js:2]: image

Comment (onionhammer) [reply src/Microsoft.Tye.Hosting/wwwroot/js/logview.js:2]: Interesting.. perhaps because of the name?

Comment (onionhammer) [reply src/Microsoft.Tye.Hosting/wwwroot/js/logview.js:2]: @jkotalik can you reproduce if you change the name to 'autoscroll.js'?

Comment (ikkentim) [reply src/Microsoft.Tye.Hosting/wwwroot/js/logview.js:2]: Privacy filter/blocker contains /logview.js as a filter https://easylist-downloads.adblockplus.org/easyprivacy.txt ;)

Comment (onionhammer) [reply src/Microsoft.Tye.Hosting/wwwroot/js/logview.js]: Good find, I renamed it to 'utilities.js'

**Comment by jkotalik** [comment]
Looks good!

**Review by jkotalik — APPROVED** [review]

**Pull request merged**